internal/controller/driver_controller.go

```diff
@@ -1157,92 +1328,6 @@ func (r *driverReconcile) reconcileNodePluginDeamonSet() error {
 								),
 							},
 						}
-						// CSI Addons Sidecar Container
-						if r.isRdbDriver() && ptr.Deref(r.driver.Spec.DeployCsiAddons, false) {
-							port := utils.NodePluginCsiAddonsContainerPort
-							containers = append(containers, corev1.Container{
-								Name:            "csi-addons",
-								Image:           r.images["addons"],
-								ImagePullPolicy: imagePullPolicy,
-								SecurityContext: &corev1.SecurityContext{
-									Privileged: ptr.To(true),
-									Capabilities: &corev1.Capabilities{
-										Drop: []corev1.Capability{"All"},
-									},
-								},
-								Args: utils.DeleteZeroValues(
-									[]string{
-										utils.CsiAddonsNodeIdContainerArg,
-										utils.LogVerbosityContainerArg(logVerbosity),
-										utils.CsiAddonsAddressContainerArg,
-										utils.ContainerPortArg(port),
-										utils.PodContainerArg,
-										utils.NamespaceContainerArg,
-										utils.PodUidContainerArg,
-										utils.StagingPathContainerArg(kubeletDirPath),
-										utils.If(logRotationEnabled, utils.LogToStdErrContainerArg, ""),
-										utils.If(logRotationEnabled, utils.AlsoLogToStdErrContainerArg, ""),
-										utils.If(logRotationEnabled, utils.LogFileContainerArg("csi-addons"), ""),
-									},
-								),
-								Ports: []corev1.ContainerPort{
-									port,
-								},
-								Env: []corev1.EnvVar{
-									utils.NodeIdEnvVar,
-									utils.PodNameEnvVar,
-									utils.PodNamespaceEnvVar,
-									utils.PodUidEnvVar,
-								},
-								VolumeMounts: utils.Call(func() []corev1.VolumeMount {
-									mounts := []corev1.VolumeMount{
-										utils.PluginDirVolumeMount,
-									}
-									if logRotationEnabled {
-										mounts = append(mounts, utils.LogsDirVolumeMount)
-									}
-									return mounts
-								}),
-								Resources: ptr.Deref(
-									pluginSpec.Resources.Addons,
-									corev1.ResourceRequirements{},
-								),
-							})
-							// Liveness Sidecar Container
```

Comment:
why liveness is removed?

Reply:
This was due to an earlier review at: https://github.com/ceph/ceph-csi-operator/pull/253#discussion_r2156584479

Reply:
Ah my bad, fixed :)